### PR TITLE
Add doctype tag and 2 Vscode snippet

### LINKED ISSUE #50: Add support for HTML attributes without any value
Generate the following HTML code:

```html
<option value="audi" selected>Audi</option>
```

the 'select' keyword has no '=' with a value. 

This should be implemented using a parameter of type [switch] on the HTML tag function.
This could need an update to Set-HTMLTag cmdlet. (To confirm)

Currently, the functions that use switches, work as followed:

```powershell
 option -Disabled
```
And return the following `HTML` code:

```HTML
<option Disabled="True"  >
</option>
```

In this case, the Disabled should be outputed, without the '="True"' part.

Readm more --> https://www.w3schools.com/tags/att_option_selected.asp

### LINKED ISSUE #101: Snippet: Create snippet basic
Create a snippet which has the following HTML code:

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
</head>
<body>
    
</body>
</html>
```

### LINKED ISSUE #108: Content on Blockquote doesnt converts to ScriptBlock
### Subject of the issue
Blockquote converts child ScriptBlock to string insteqd of scriptblock

### Steps to reproduce
```powershell
blockquote -Content { p{"woop"}}
 
#or

blockquote { p {woop}}

```

### Expected behaviour
```html
<blockquote>
 <p>
  "woop"
 </p>
</blockquote>
```
### Actual behaviour
```html
<blockquote>
 p{"woop"}
</blockquote>
```

### PR DESCRIPTION
**Pull request by christophekumor**
# Pull Request 

- Add doctype tag function
- Add 2 Vscode Snippets
- Add a ValidateSet on meta function
- Fix some double quote's escape not needed
- Mod $htmltagparams filling to make it universal
- Fix some pester test

### Please tell us , the type of Change you are submiting:
Select one of the following: 

- [x] Bug
- [x] Feature
- [X] Minor Change

### Does it fix an existing issue? Please tell us which one

Resolves #108
Resolves #101 
Resolves #50
Resolves Part #61

### CODE REVIEW

**Comment by Stephanevg** [comment]
Hi @christophekumor Thank you SO much for this PR

I actually would have accepted the tags PR's if I could seperatley, already, because they seem really good. Thanks a lot for that! 👍

Concerning the change on the 50 files, I have difficulties to understand the issue you are resolving / fiying with this. Was this tracked in a specific issue n°?

It would be great if you can show the problem you faced, with the older version. (And that, should actually become a new pester test btw ;))

**Comment by christophekumor** [comment]
Voici mes tests Perster sur ta branch DEV
là ça met en avant l'absence des param par défaut qui ne sont pas pris en compte.

File: PSHTML/Functions/Public/base.ps1 [hunk 1]
```diff
@@ -42,34 +43,58 @@ Function base {
...
-                if($entry -eq "content"){
-
-
-                    $htmltagparams.$entry = $PSBoundParameters[$entry]
-                }else{
-                    $htmltagparams.$entry = "{0}" -f $PSBoundParameters[$entry]
+    }
+    Process {
+        foreach ($paramkey in $MyInvocation.MyCommand.Parameters.Keys) {
+            $paramvalue = Get-Variable $paramkey -ValueOnly -EA SilentlyContinue
```
Comment (Stephanevg) [inline PSHTML/Functions/Public/base.ps1]: Hi, You could change it like this:

```suggestion
            If($paramkey -and !$PSBoundParameters.ContainsKey($paramkey)){
                   $htmltagparams.$paramkey = $paramvalue
            }
```

Comment (christophekumor) [reply PSHTML/Functions/Public/base.ps1]: as see with you, it's not possible.

**Pull request closed without merging**